packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -613,7 +617,7 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
       verboseName: verboseName ?? this.verboseName,
       category: category ?? this.category,
       rawUrl: url ?? rawUrl,
-      processedUrl: url != null ? getSimplePackageUrl(url) : processedUrl,
+      processedUrl: processedUrl ?? this.processedUrl,
       sourceLine: sourceLine,
```

Comment:
again, not your fault, but while you're here could you add the missing param for sourceLine and use it here `sourceLine ?? this.sourceLine`

Reply:
Going to handle this in https://github.com/flutter/devtools/issues/3974